content/md/productivity/get-started-with-the-rules-engine.md

```diff
@@ -0,0 +1,656 @@
+
+```YML
+  actions:
+      type: remove
+        field: categories
+        items:
+          - t-shirts
+```
+
+To remove the “clothing” category and its children, the action will be as follows:
+```YML
+  actions:
+      type: remove
+      field: categories
+      items:
+        - clothing
+      include_children: true
+```
+
+To unclassify products from the whole “Master catalog” tree, the action will be as follows:
+
+```YML
+  actions:
+      type: remove
+      field: categories
+      items:
+        - master
+      include_children: true
+```
+
+## Concatenate
+
+The `Concatenate` action concatenates at least two values into a single value. A space separates each source value.
+
+The possible source attribute types are:
+ - text
+ - text area
+ - date
+ - identifier
+ - metric
+ - number
+ - price collection
+ - simple or multi select (option codes)
+
+The possible target attribute types are:
+ - text
+ - text area
+
+**The parameters `from` and `to` are required in this format. Depending on the source attribute type, some optional keys can be added.**
+
+The expected values are:
+
+**`from`**
+- `field`: the attribute code.
+- `locale`: the locale code for which the value is assigned (optional)
+- `scope`: the channel code for which the value is assigned (optional)
+- For `date` attribute: the format of the date following the PHP format [specification](https://www.php.net/manual/en/function.date.php) (Optional). By default, it is *Y-m-d* (e.g. *2020-01-31*)
+- For `price collection` attribute: the currency code for which the price is assigned (Optional). By default all the prices in the collection are displayed, separated by a coma.
+
+**`to`**
+- `field`: the attribute code.
+- `locale`: the locale code for which the value is assigned (optional)
+- `scope`: the channel code for which the value is assigned (optional)
+
+### Example
+
+To concatenate the **brand** (non localizable and non scopable) and the **model** in the `en_US` locale into the **description** value in the `en_US` locale, the action will be as follows:
+
+```YML
+  actions:
+    - type: concatenate
+        from:
+            field: brand
+            field: model
+            locale: en_US
+        to:
+            field: description
+            locale: en_US
+```
+
+To concatenate the **model** in the `en_US` locale, the **color** in the `en_US` locale and the **year of the release date** into the **title** value in the `en_US` locale, the action will be as follows:
+
+```YML
+  actions:
+    - type: concatenate
+        from:
+            field: model
+            locale: en_US
+            field: color
+            locale: en_US
+            field: release_date
+            format: Y
+        to:
+            field: title
+            locale: en_US
+```
+
+To concatenate the **model** in the `en_US` locale and the **price** in USD from the `mobile` channel into the **subtitle** value in the `en_US` locale and the `mobile` channel, the action will be as follows:
```

Comment:
Examples look weird to me. Do you feel they match customer use cases?

Reply:
honestly, I didn't change the examples :D I copy pasted the content of nini. But yes, I'll change to make it more user friendly

Reply:
Actually we can remove this example, we already have 2 examples for this action, whereas all the other ones have only 1 example. Maybe we can remove 2 of 3 examples. What do you think?